Specs/DateTime/English/MergedParser.json

```diff
@@ -309,29 +309,6 @@
 }
 ,
 {
-  "Input": "put make cable's wedding in my calendar for wednesday the thirty first",

```

Comment:
Why was this removed?

Reply:
added back. The output is different since September doesn't contain the thirty first.